DOCKER.md

```diff
@@ -25,19 +25,19 @@ you and your users can direct s3 server requests to example.com.
 docker run -d --name s3server -p 8000:8000 -e HOST_NAME=new.host.com scality/s3server
 ```
 
-#### ACCESS_KEY and SECRET_KEY
+#### SCALITY_ACCESS_KEY_ID and SCALITY_SECRET_ACCESS_KEY
 
 These variables specify authentication credentials for an account
-named "Docker".
+named "CustomAccount".
 
 You can set credentials for many accounts by editing `conf/authdata.json`
 (see below for further info),
 but if you just want to specify one set of your own,
 you can use these environment variables.
```

Comment:
Please add here as well -- Note that if you set these environment variables, the conf/authdata.json will be ignored.